packages/client/src/Authentication.ts

```diff
@@ -15,18 +14,13 @@ export interface Authentication {
     // always in lowercase
     getAddress: () => Promise<EthereumAddress>
     signWithWallet: (payload: Uint8Array) => Promise<Uint8Array>
```

Comment:
Maybe the `withWallet` suffix is not important as there is no other counterpart to distinguish from?  

Also, would it make just expose one `getSigner` function instead of exposing `getStreamRegistryChainSigner` and `signWithWallet`?

```suggestion
    signWithWallet: (payload: Uint8Array) => Promise<Uint8Array>
```